src/libsync/propagateremotemkdir.cpp

```diff
@@ -158,11 +158,18 @@ void PropagateRemoteMkdir::success()
         connect(watcher, &QFutureWatcherBase::finished, this, [watcher, this] {
             const auto *result = watcher->result().get();
             if (*result) {
-                Q_ASSERT(result->get() == Vfs::ConvertToPlaceholderResult::Ok);
-                done(SyncFileItem::Success);
-            } else {
-                done(SyncFileItem::FatalError, result->error());
+                switch (result->get()) {
+                case Vfs::ConvertToPlaceholderResult::Ok:
+                    done(SyncFileItem::Success);
+                    return;
+                case Vfs::ConvertToPlaceholderResult::Locked:
+                    done(SyncFileItem::SoftError, tr("UpdateMetadata Timed out"));
```

Comment:
we could go for a more friendly string, such as "Set file status failed due to file lock." maybe? Is the error displayed together with the file name?

Reply:
yes and I'll change the description